### ISSUE TITLE
How to install GPyOpt in Windows 10

### ISSUE THREAD

**Issue by debipe20** [SheffieldML/GPyOpt#110]
I was trying to install GPyOpt package by using pip command at my python 27/scripts folder. But it doesn't work. It states the following:

from numpy._distributor_init import NUMPY_MKL # requires numpy+mkl

ImportError: cannot import name NUMPY_MKL

Command "python setup.py egg_info" failed with error code 1 in c:\users\debip\appdata\local\temp\pip-build-rsio3n\gpyopt\
C:\Python27\Scripts>

Can you suggest me something? I am using python 2.7(32bit). I will also like to link GPyOpt with anaconda

**Comment by tinutjoy** [comment]
Here is a pointer which might help you, https://stackoverflow.com/questions/37267399/importerror-cannot-import-name-numpy-mkl. I have installed GPyOpt in windows 10 under Ananaconda python 3.5 without any problem.

**Comment by apaleyes** [comment]
Hey @debipe20 , did you have any luck following the tip from @tinutjoy ?

**Comment by debipe20** [comment]
Hello,
I have tried it but unfortunately it didn't work for me.



*Regards,*
*Debashis Das*
*Graduate Assistant*
*Systems and Industrial Engineering*
*The University of Arizona*

**Comment by tinutjoy** [comment]
Hey @debipe20 ,

Are you using anaconda environment? If you are using anaconda, create a python 3.x virtual environment under it and install the package there.  

However, I have installed it on both 2.7 and 3.x.  

Cheers,
Tinu

**Comment by debipe20** [comment]
Currently, I am using python 2.7. I will try one more time. Thanks for the
assistance.



*Regards,*
*Debashis Das*
*Graduate Assistant*
*Systems and Industrial Engineering*
*The University of Arizona*

**Comment by javiergonzalezh** [comment]
Hi @debipe20, did you managed to install?

**Comment by XRarach** [comment]
Hi, do you have these 3 dependencies already installed? 

GPy (>=1.0.8)
numpy (>=1.7)
scipy (>=0.16)

pip install GPy 
pip install numpy 
pip install scipy

**Comment by debipe20** [comment]
Yes, I have installed GPy (>=1.0.8) numpy (>=1.7) scipy (>=0.16).



*Regards,*
*Debashis Das*
*Graduate Assistant*
*Systems and Industrial Engineering*
*The University of Arizona*
